<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkloadStatistics</class>
 <widget class="QDialog" name="WorkloadStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Статистика</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pB_close">
           <property name="text">
            <string>Закрыть</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabW_monthly_statistics">
          <attribute name="title">
           <string>За месяц</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QCustomPlot" name="wd_monthly_statistics" native="true"/>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="cB_month"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabW_statistics_for_the_year">
          <attribute name="title">
           <string>За год</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QCustomPlot" name="wd_statistics_for_the_year" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lb_airportName">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Аэропорт</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>C:\kurs_cpp\Airport_inspector\Airport_inspector\qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pB_close</sender>
   <signal>clicked()</signal>
   <receiver>WorkloadStatistics</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
